docs/lib/graphqlapi/fragments/flutter/advanced-workflows/10_example.md

```diff
@@ -0,0 +1,28 @@
+```dart
+try{
+    String graphQLDocument = '''mutation createTodo(\$input: CreateTodoInput!) {
+      createTodo(input: \$input) {
+        id
+        name
+        description
+      }
+    }''';
+
+    var operation = await Amplify.API.mutate(
```

Comment:
Why do we need to await here? Don't all API methods return `Operation` synchronously and the response inside the operation is asynchronous?

Reply:
Good point, removed await from the examples.